<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPingForm</class>
 <widget class="QWidget" name="AddPingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_save">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_qci">
        <property name="text">
         <string>qci</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_numberofpagins">
        <property name="text">
         <string>numberofpagins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_interval">
        <property name="text">
         <string>Interval</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_minRecievedPings">
        <property name="text">
         <string>minRecievedPings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="qciComboPing"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tet_numberOfPagins"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tet_interval"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tet_minRecievedPings"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
